### PR TITLE
Chinese Translations

### PR DESCRIPTION
**Pull request by dianliang233**
1. Delete Chinese language
2. ...and replacing it with 2 variant: Simplified (`zh-hans`) and Traditional (`zh-hant`)
3. New translations

This is a breaking change, you should replace the current usage of `zh` with `zh-hans` (recommended, because OG `zh` is based on `zh-hans`).

### CODE REVIEW

File: i18n/allLangs.json [hunk 1]
```diff
@@ -36,9 +37,17 @@
...
+			"chinese": "zh-hans",
+			"中文": "zh-hans",
+			"zh-hans": "zh-hans",
+			"Chinese (Simplified)": "zh-hans",
+			"Simplified Chinese": "zh-hans",
+			"简体中文": "zh-hans",
+			"zh-hant": "zh-hant",
+			"Chinese (Traditional)": "zh-hant",
+			"Traditional Chinese": "zh-hant",
+			"繁體中文": "zh-hant"
```
Comment (Markus-Rost) [inline i18n/allLangs.json:50]: the keys should be lower case only. 

Feel free to add more possible names for the language. those keys are what users can enter to get the language.

**Comment by dianliang233** [comment]
> You might want to add the plural function in some strings: `{{PLURAL:number|singular|plural}}`
> See [en.json](https://github.com/Markus-Rost/discord-wiki-bot/blob/master/i18n/en.json#L335-L337) for examples.

Actually most Chinese words have no plural forms.

**Comment by dianliang233** [comment]
> In the `rcscript` string you are referring to RSS. It might have been a bit unclear but "feeds" in the English version refers to the code [discussions](https://community.fandom.com/wiki/Help:Discussions) is based on. Example: https://wikibot.fandom.com/f

This word is pretty hard to translate and I've been discussing this with folks from zh.community.

**Comment by Markus-Rost** [comment]
> Actually most Chinese words have no plural forms.
Ok, I didn't know that, just wanted to make sure.

> This word is pretty hard to translate and I've been discussing this with folks from zh.community.
You could use "discussions" instead of "feeds", that should be easier to translate.

**Pull request merged**